Documentation/Settings.cfg

```diff
@@ -21,20 +21,20 @@ project_discussions  =
 project_home         =
 project_issues       = https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi/issues
 project_repository   = https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-CoreApi
-use_opensearch       = https://docs.typo3.org/m/typo3/reference-coreapi/master/en-us/
+use_opensearch       = https://docs.typo3.org/m/typo3/reference-coreapi/10.4/en-us/
```

Comment:
These changes - though nessesary - should be made in a separate PR

Reply:
A separate commit should be enough, not?